src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -19,17 +19,17 @@
 public class SampleDataUtil {
     public static Profile[] getSampleProfiles() {
         return new Profile[] {
-            new Profile(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
+            new Profile(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@u.nus.edu"),
                 new Telegram("lexyeoh"), getTagSet("friends")),
-            new Profile(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
+            new Profile(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@u.nus.edu"),
```

Comment:
Very minor, but might want to include the different possible domains in the sample?